sdk/enoki/src/wallet/index.ts

```diff
@@ -0,0 +1,341 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { SuiClient } from '@mysten/sui/client';
+import { Transaction } from '@mysten/sui/transactions';
+import type {
+	IdentifierArray,
+	StandardConnectFeature,
+	StandardConnectMethod,
+	StandardDisconnectFeature,
+	StandardDisconnectMethod,
+	StandardEventsFeature,
+	StandardEventsListeners,
+	StandardEventsOnMethod,
+	SuiSignAndExecuteTransactionFeature,
+	SuiSignAndExecuteTransactionMethod,
+	SuiSignPersonalMessageFeature,
+	SuiSignPersonalMessageMethod,
+	SuiSignTransactionFeature,
+	SuiSignTransactionMethod,
+	Wallet,
+} from '@mysten/wallet-standard';
+import { getWallets, ReadonlyWalletAccount } from '@mysten/wallet-standard';
+import type { Emitter } from 'mitt';
+import mitt from 'mitt';
+
+import { fromB64, toB64 } from '../../../bcs/src/b64.js';
+import type { EnokiNetwork } from '../EnokiClient/type.js';
+import type { AuthProvider, EnokiFlowConfig } from '../EnokiFlow.js';
+import { EnokiFlow } from '../EnokiFlow.js';
```

Comment:
I think right now it's fine that this uses EnokiFlow, but I'd love to actually kill this and just have this be implemented here + deprecate flow

Reply:
deprecated EnokiFlow, we can fully remove it in the future